webpack.config.js

```diff
@@ -85,6 +85,7 @@ const entryPointNames = [
 const packageNames = [
 	'hooks',
 	'i18n',
+	'is-shallow-equal',
```

Comment:
Should we do the same with `wordcount`, `dom-ready` and others? I think `wordcount` is included in `wp.utils` and `dom-ready` is included in `edit-post`. I'm thinking how we should we approach those packages in general.

Reply:
>Should we do the same with `wordcount`, `dom-ready` and others? I think `wordcount` is included in `wp.utils` and `dom-ready` is included in `edit-post`. I'm thinking how we should we approach those packages in general.

I expect for compatibility's sake we'd not want to remove any of the existing scripts, though my preference moving forward would be that there is a corresponding script handle for every single NPM package under the WordPress scope. For example:

```
@wordpress/is-shallow-equal (npm package)
wp-is-shallow-equal (script handle)
wp.isShallowEqual (browser global)
```

```
@wordpress/wordcount (npm package)
wp-wordcount (script handle)
wp.wordcount (browser global)
```

```
@wordpress/dom-ready (npm package)
wp-dom-ready (script handle)
wp.domReady (browser global)
```

Reply:
I like this idea. As soon as this is merged into core, it should be even easier to maintain those small dependencies, because they will change only between the releases so you will be able to batch them together as long as HTTP/1 is the thing.